Doc/library/mimetypes.rst

```diff
@@ -52,7 +52,22 @@ the information :func:`init` sets up.
    are also recognized.
 
    .. versionchanged:: 3.8
-      Added support for url being a :term:`path-like object`.
+      Added support for *url* being a :term:`path-like object`.
+
+   .. versionchanged:: 3.13
+      Passing a file path instead of URL is deprecated.
```

Comment:
Since it's not actively deprecated, I suggest to only mention a *soft* deprecation:

```suggestion
      Passing a file path instead of URL is :term:`soft deprecated`.
```